charts/umbrella/values.yaml

```diff
@@ -639,6 +640,125 @@ iatpmock:
           - path: /
             pathType: ImplementationSpecific
 
+ssi-credential-issuer:
+  enabled: false
+  portalBackendAddress: "http://portal-backend.tx.test"
+  walletAddress: "http://managed-identity-wallets.tx.test"
+  walletTokenAddress: "http://centralidp.tx.test/auth/realms/CX-Central/protocol/openid-connect/token"
+
+  issuer:
+    swaggerEnabled: true
+    logging:
+      businessLogic: "Debug"
+      default: "Debug"
+    portal:
+      # -- Provide portal client-id from CX IAM centralidp.
+      # You must specify the technical user with the required roles for the interaction with the portal
+      clientId: "sa-cl24-01"
```

Comment:
I just checked again, and that's the correct clientId 👍 (sorry for the confusion in the call earlier)

Reply:
thanks for the review